examples/virtual_keyboard.rs

```diff
@@ -1,13 +1,14 @@
 // Create a virtual keyboard, just while this is running.
 // Generally this requires root.
 
-use evdev::{uinput::VirtualDeviceBuilder, AttributeSet, EventType, InputEvent, Key};
+use evdev::KeyEvent;
+use evdev::{uinput::VirtualDeviceBuilder, AttributeSet, EventType, InputEvent, KeyType};
 use std::thread::sleep;
 use std::time::Duration;
 
 fn main() -> std::io::Result<()> {
-    let mut keys = AttributeSet::<Key>::new();
-    keys.insert(Key::BTN_DPAD_UP);
+    let mut keys = AttributeSet::<KeyType>::new();
```

Comment:
`AttributeSet::from(KeyType::BTN_DPAD_UP)` might be a cool constructor/trait impl.